tests/SpiffWorkflow/bpmn/TooManyLoopsTest.py

```diff
@@ -1,86 +0,0 @@
-# -*- coding: utf-8 -*-
-
```

Comment:
Seems like this would be a good test to keep around -- assuming that the loop reset task is indeed no longer needed, then this test should pass without issue. Correct?

Reply:
I used a serialized version of this workflow in the test for removing these tasks so in the migration test, the loop is removed and the workflow runs to completion.  Also we have this test, which makes sure it is possible to limit task data size:https://github.com/sartography/SpiffWorkflow/blob/42b483054b3f1b86f7d3a9eb1a193345c578d86d/tests/SpiffWorkflow/bpmn/PythonScriptEngineEnvironmentTest.py#L30